message_ix_models/model/material/util.py

```diff
@@ -366,7 +373,18 @@ def path_fallback(context_or_regions: Union[Context, str], *parts) -> Path:
     raise FileNotFoundError(candidates)
 
 
-def get_pycountry_iso(row, mis_dict):
+def get_pycountry_iso(row: str, mis_dict: dict[str, str]) -> str:
```

Comment:
I suspect this can be replaced with [`.util.pycountry.iso_3166_alpha_3()`](https://docs.messageix.org/projects/models/en/latest/api/util.html#module-message_ix_models.util.pycountry)